Source/levels/themes.cpp

```diff
@@ -577,14 +574,26 @@ void Theme_MonstPit(int t)
 	PlaceThemeMonsts(t, monstrnd[leveltype - 1]);
 }
 
+namespace {
+void SpawnObjectOrSkeleton(unsigned frequency, _object_id objectType, Point tile)

```

Comment:
Looks like `objectType` is alwasy `OBJ_BANNERL`

Reply:
Shouldn't be, it's left right middle middle right left

Reply:
Are you trying to use the Konami Code on me :eyes: